iati/tests/data/activities.json

```diff
@@ -0,0 +1 @@
ay_name": "11684", "name": "11684"}, {"count": 1, "display_name": "1162", "name": "1162"}, {"count": 3, "display_name": "116", "name": "116"}, {"count": 1, "display_name": "1151", "name": "1151"}, {"count": 6, "display_name": "115", "name": "115"}, {"count": 1, "display_name": "1148", "name": "1148"}, {"count": 1, "display_name": "1146", "name": "1146"}, {"count": 3, "display_name": "114", "name": "114"}, {"count": 1, "display_name": "1139", "name": "1139"}, {"count": 1, "display_name": "1138", "name": "1138"}, {"count": 4, "display_name": "113", "name": "113"}, {"count": 1, "display_name": "1129", "name": "1129"}, {"count": 6, "display_name": "112", "name": "112"}, {"count": 1, "display_name": "1115", "name": "1115"}, {"count": 6, "display_name": "111", "name": "111"}, {"count": 1, "display_name": "1109", "name": "1109"}, {"count": 1, "display_name": "1101", "name": "1101"}, {"count": 4, "display_name": "110", "name": "110"}, {"count": 80, "display_name": "11", "name": "11"}, {"count": 1, "display_name": "1091", "name": "1091"}, {"count": 1, "display_name": "1090", "name": "1090"}, {"count": 5, "display_name": "109", "name": "109"}, {"count": 1, "display_name": "1089", "name": "1089"}, {"count": 2, "display_name": "108", "name": "108"}, {"count": 1, "display_name": "1079", "name": "1079"}, {"count": 1, "display_name": "1078", "name": "1078"}, {"count": 1, "display_name": "1076", "name": "1076"}, {"count": 1, "display_name": "1071", "name": "1071"}, {"count": 1, "display_name": "1070", "name": "1070"}, {"count": 7, "display_name": "107", "name": "107"}, {"count": 1, "display_name": "1069", "name": "1069"}, {"count": 1, "display_name": "1067", "name": "1067"}, {"count": 2, "display_name": "1066", "name": "1066"}, {"count": 5, "display_name": "106", "name": "106"}, {"count": 1, "display_name": "1053", "name": "1053"}, {"count": 8, "display_name": "105", "name": "105"}, {"count": 2, "display_name": "1046", "name": "1046"}, {"count": 1, "display_name": "1041", "name": "1041"}, {"count": 4, "display_name": "104", "name": "104"}, {"count": 1, "display_name": "1035", "name": "1035"}, {"count": 2, "display_name": "103", "name": "103"}, {"count": 1, "display_name": "1027", "name": "1027"}, {"count": 1, "display_name": "1024", "name": "1024"}, {"count": 1, "display_name": "1022", "name": "1022"}, {"count": 1, "display_name": "10216", "name": "10216"}, {"count": 3, "display_name": "102", "name": "102"}, {"count": 1, "display_name": "10122", "name": "10122"}, {"count": 2, "display_name": "1012", "name": "1012"}, {"count": 1, "display_name": "1011", "name": "1011"}, {"count": 5, "display_name": "101", "name": "101"}, {"count": 1, "display_name": "1002", "name": "1002"}, {"count": 1, "display_name": "1001", "name": "1001"}, {"count": 2, "display_name": "100", "name": "100"}, {"count": 107, "display_name": "10", "name": "10"}, {"count": 1096, "display_name": "1", "name": "1"}, {"count": 4, "display_name": "0", "name": "0"}], "title": "extras_activity_count"}}}}
```

Comment:
Great that we've added test data files in their own folder. 

As a note, typically these files tend to include a minimal subset of expected data (and be pretty formatted) so that you can see the expected structure of the file.